src/main/groovy/com/netflix/gradle/plugins/utils/FilePermissionUtil.groovy

```diff
@@ -14,6 +14,11 @@ import org.gradle.api.file.SyncSpec
 @CompileDynamic
 class FilePermissionUtil {
 
+    private static final int DEFAULT_FILE_PERMISSION = 0644  // rw-r--r-- (420 in decimal)
```

Comment:
Could use https://docs.oracle.com/javase/8/docs/api/java/nio/file/attribute/PosixFilePermissions.html here, it'll take the string version of these.